server/operator.go

```diff
@@ -406,7 +437,9 @@ func newTransferLeaderOperator(regionID uint64, oldLeader, newLeader *metapb.Pee
 }
 
 func (op *transferLeaderOperator) String() string {
-	return fmt.Sprintf("%+v", *op)
+	op.RLock()
+	defer op.RUnlock()
+	return jsonString(op)
```

Comment:
operators would not be performance bottleneck, so I prefer readability to performance here.